### PR TITLE
Update issue templates

### LINKED ISSUE #1: Make Feature Request Issue Template
Deliverables: Make a nice issue template for the project.

How to make one ? [Resource](https://docs.github.com/en/github/building-a-strong-community/configuring-issue-templates-for-your-repository)

- Content to be added:
**Is your feature request related to a problem? Please describe.**
**Describe the solution you'd like**
**Describe alternatives you've considered**
**Additional context**

### PR DESCRIPTION
**Pull request by SangeetaMishra**
Added the Feature Request Issue Template

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: ''
+labels: ''
+assignees: ''
```
Comment (ghost) [inline .github/ISSUE_TEMPLATE/feature_request.md]: Please remove these three unnecessary things as they are blank.

**Review by ghost — COMMENTED** [review]
Please do the requested changes @SangeetaMishra

**Comment by SangeetaMishra** [comment]
@urvashi-code1255 I have made the changes

**Review by ghost — APPROVED** [review]
Good work @SangeetaMishra !!

**Pull request merged**